packages/pluto/readme.md

```diff
@@ -8,16 +8,24 @@ Pluto is an e2e testing framework that uses Selenium to execute test cases. It i
 
 ## Installation
 
-- Install Pluto as a global npm package.
+### Install Pluto as a global npm package.
 ```shell
 npm install -g @creately/pluto
 ```
 
-- Set up Chrome and Firefox webdrivers for Selenium
+Note: When installing the package globally, your $NODE_PATH environment variable should be set to the global package directory. If it isn't set already, add the following to .bashrc or .zshrc :
```

Comment:
Why do we need to set the `NODE_PATH` env variable?

Reply:
So that node will also check the global directory for packages, otherwise it will try the local `node_modules` and fail when the module is required. There may be a better way to get this done though, which I was thinking of adding to another story.

Reply:
okay